### DISCUSSION TITLE
review classification of Hereditary Hemochromatosis

### DISCUSSION THREAD

**Discussion by nicolevasilevsky** [monarch-initiative/mondo#3866] [Bigger project]
https://github.com/monarch-initiative/helpdesk/issues/31

**Comment by nicolevasilevsky** [comment]
Note to self: need to look into hemochromatosis classification further and flesh out our hierarchy eg bronze diabetes.

the parent of 'hereditary hemochromatosis' is hemosiderosis, we don't have a general hemochromatosis class. I'll look into this further

**Comment by nicolevasilevsky** [comment]
from Peter:
Many but not all cases of hemochromatosis are Mendelian.

**Comment by ValWood** [comment]
Heamochromatosis shouldnt be is_a Hemosiderosis

Difference between haemochromatosis and Hemosiderosis
Hemochromatosis is a rare disease in which direct deposition of iron takes place in body organs such as liver, spleen, pancreas and skin that cause tissue damage. Hemosiderosis on the other hand side is a disorder characterized by deposition of excess iron within the body tissues that normally do not containing iron.

**Comment by nicolevasilevsky** [comment]
Thanks @ValWood!
